app/views/entities/index.html.erb

```diff
@@ -20,7 +20,7 @@
 </div>
 <% end %>
 <% else %>
-<p>No Transactions</p>
+<p class="text-light">No Transactions</p>
```

Comment:
- [ ] Please kindly ensure that the categories and transactions pages match the below design.
- [ ] Also, kindly ensure that other pages follow the design guidelines.

![image](https://user-images.githubusercontent.com/43995830/203873172-3de2c41e-b969-4c92-83ad-97a802099c98.png)

Reply:
Done ✔️